### PR TITLE
create stub file for KC

### PR DESCRIPTION
**Pull request by tjroberts**
Just trying this out!

### CODE REVIEW

File: tyler-kc.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# San Diego (NOT KC!)
+
+#Places to eat
+- Phil's BBQ!
```
Comment (vp186006) [inline tyler-kc.md:4]: Like it! We're going there!

Comment (eminbagh) [reply tyler-kc.md:4]: @vp186006 It's okay. I like Karl Strauss more!

Comment (tjroberts) [reply tyler-kc.md:4]: Waygu burger sounds good right now...

Comment (eminbagh) [reply tyler-kc.md:4]: @tjroberts @vp186006 Anything sounds good right now. specially, a TD cafe chicken sandwich. I'm starving!

**Pull request merged**